make/run_make.mk

```diff
@@ -40,3 +40,6 @@ plain smp frag smp_frag:
 clean generate depend docs release release_spec release_docs release_docs_spec \
   tests release_tests release_tests_spec static_lib xmllint:
 	$(make_verbose)$(MAKE) -f $(TARGET)/Makefile $@
+
+test:
```

Comment:
Do you need the test target in here?

Reply:
No, that is a mistake. I will remove that. Thanks!